src/js/plugins/plugin.mobile_nav.js

```diff
@@ -121,9 +122,14 @@ BookReader.prototype.initToolbar = (function (super_) {
           $mmenuEl.data('mmenu').close();
         } else {
           $mmenuEl.data('mmenu').open();
+          this.trigger("mobileNavOpen");
         }
       });
 
+      $(".mm-next.mm-fullsubopen[data-target='#mm-4']").click(
+        () => this.trigger("TOCOpen")
+      );
```

Comment:
mobile_nave.js shouldn't know that a TOC is inside it; let's see if we can avoid this event altogether. Let's make plugin.chapters.js bind to the click event directly.

```suggestion
```

Also: avoid using `mm-4`; it'll change if more mobile drawer elements are added. Something like `.BRmobileMenu__tableContents > .mm-fullsubopen` should do the trick.

Reply:
done, also removed TOCOpen event